### ISSUE TITLE
renaming python install folder or exe upsets running analyzer.py?

### ISSUE THREAD

**Issue by mallorybobalice** [cuckoosandbox/cuckoo#2859]
If I'm not mistaken, 
the path is hardcoded here. 

https://github.com/cuckoosandbox/cuckoo/blob/13cbe0d9e457be3673304533043e992ead1ea9b2/cuckoo/core/guest.py#L524

I rather like what they did here with inheriting the agent py path 
https://github.com/ctxis/CAPE/blob/3a3c2e0dc2633b5c6f901ffabf4bd54235c581cd/agent/agent.py#L180

thoughts?

**Comment by mallorybobalice** [comment]
note, in cape, the invoking of analyzer is in agent.py not guest.py ... i think maybe.

**Comment by doomedraven** [comment]
you are wrong, you are in cuckoo v2 aka upstream and posting stuff from CAPE

**Comment by mallorybobalice** [comment]
=) the second link is cape, the first is cuckoo2 .
 I have both installed in different venv/pointing to different dbs.

**Comment by mallorybobalice** [comment]
I'm fairly sure it's not a problem for cape (inherits agent.py path) and is a problem for cuckoo 2.0.7 (hardcoded path).

**Comment by doomedraven** [comment]
ah damn my fail sorry i though github just extended link :D, well you can change that in your side, as you can see PRs are not acepted for years already

**Comment by mallorybobalice** [comment]
m I've fixed the hardcoded path for cuckoo 2 but it's a pita to reconcile 1 more python file on updates

**Comment by doomedraven** [comment]
yep i know, you always can fork and just apply the changes from upstream is what i was using when was running linux cuckoo

**Comment by mallorybobalice** [comment]
mm I noticed git ui sorta shows you the changes vs fork... 

penny for your thoughts on cape vs cuckoo2 . I see you forked both :D ... or is that to track changes vs your ones?
(and used to look after cuckoomodified/spendersandbox for a bit after Brad retired from it)

**Comment by doomedraven** [comment]
im personally using cape due to better injection tracing and many other things, as for the sadness stuff here isn't merged anymore and cherry picks all the stuff, you can get much better v2 if you pay for it, but opensource is kinda abondoned at the moment :(

**Comment by mallorybobalice** [comment]
icic . thank you . 

> better v2 if you pay for it

:\ thewhaaat. please elaborate?

**Comment by doomedraven** [comment]
https://hatching.io/solutions
